stardis/plasma/base.py

```diff
@@ -184,16 +178,17 @@ def create_splasma(marcs_model_fv, marcs_abundances_all, adata, tracing_nus):
     plasma_modules.append(HMinusDensity)
     plasma_modules.append(TracingNus)
```

Comment:
I think we can also remove `TracingNus` here as well as the class above but other than that this is perfect. You reran the notebook right? Just to double check that everything still works.

Reply:
I tried and it gives:
```
PlasmaMissingModule: Trying to update property tracing_nus that is unavailable
```